test/livebook_teams/hubs/team_client_test.exs

```diff
@@ -656,9 +657,13 @@ defmodule Livebook.Hubs.TeamClientTest do
       # Since the app deployment struct generation is from Livebook side,
       # we don't have yet the information about who deployed the app,
       # so we need to add it ourselves.
+      {seconds, 0} = NaiveDateTime.to_gregorian_seconds(teams_app_deployment.updated_at)
+
       app_deployment = %{
         app_deployment
         | id: to_string(teams_app_deployment.id),
+          version:
+            "#{teams_app_deployment.id}-#{teams_app_deployment.app_revision.sha}-#{seconds}",
```

Comment:
FWIW now the version is opaque, so you can totally put `Livebook.Utils.random_id()` here or similar (unless this breaks something (which it should not)).